e2e-tests/tests/committee/test_multisig_governance.py

```diff
@@ -135,14 +135,20 @@ def test_verify_multisig_policy(api: BlockchainApi, additional_governance_author
 
 @mark.xdist_group(name="governance_action")
 @mark.usefixtures("governance_skey_with_cli")
-def test_multisig_upsert_d_parameter(api: BlockchainApi, governance_authority, additional_governance_authorities):
+def test_multisig_upsert_d_parameter(
+    api: BlockchainApi, current_mc_epoch, governance_authority, additional_governance_authorities
+):
     """Test a multisig operation that modifies the D parameter"""
     # Try to update D parameter
 
     # TODO: Extract logic to calculate unique DParam
+    current_d_param = api.get_d_param(current_mc_epoch + 2)
```

Comment:
Could you take care for this TODO above. It seems that logic gets complicated just enough for a separate method.

Reply:
Sure thing. I'll do it in ETCM-9682